webapp/store/index.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {batchActions} from 'redux-batched-actions';
+import {configureOfflineServiceStore} from 'mattermost-redux/store';
+import {General, RequestStatus} from 'mattermost-redux/constants';
+import reduxInitialState from 'mattermost-redux/store/initial_state';
+import {createTransform, persistStore} from 'redux-persist';
+import localForage from 'localForage';
+
+import {transformSet} from './utils';
+
+const usersSetTransform = [
+    'profilesInChannel',
+    'profilesNotInChannel',
+    'profilesInTeam',
+    'profilesNotInTeam'
+];
+
+const teamSetTransform = [
+    'membersInTeam'
+];
+
+const setTransforms = [
+    ...usersSetTransform,
+    ...teamSetTransform
```

Comment:
Not sure what you mean, is there a new way to do this now? The RN repo looks like it's doing the same way unless I'm missing something